examples/fluids/src/grid_anisotropy_tensor.c

```diff
@@ -166,11 +151,13 @@ PetscErrorCode GridAnisotropyTensorCalculateCollocatedVector(Ceed ceed, User use
   CeedOperatorCreate(ceed, qf_colloc, NULL, NULL, &op_colloc);
   CeedOperatorSetField(op_colloc, "qdata", ceed_data->elem_restr_qd_i, CEED_BASIS_COLLOCATED, ceed_data->q_data);
   CeedOperatorSetField(op_colloc, "v", *elem_restr_grid_aniso, CEED_BASIS_COLLOCATED, CEED_VECTOR_ACTIVE);
-  CeedOperatorSetNumQuadraturePoints(op_colloc, CeedIntPow(num_qpts_1d, dim));
+  CeedOperatorSetNumQuadraturePoints(op_colloc, num_qpts);
 
   CeedElemRestrictionCreateVector(*elem_restr_grid_aniso, aniso_colloc_ceed, NULL);
 
   CeedOperatorApply(op_colloc, CEED_VECTOR_NONE, *aniso_colloc_ceed, CEED_REQUEST_IMMEDIATE);
 
+  CeedQFunctionDestroy(&qf_colloc);
+  CeedOperatorDestroy(&op_colloc);
```

Comment:
This is also a change - should we do some scrubbing of the fluids app for memory leaks? I thought we had those checks in CI?

Reply:
We definitely should do some scrubbing. I'm not sure if we got memory leak checks into CI.

Reply:
(fixed in #1266)